frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -540,35 +539,12 @@ export const ADMIN_SETTINGS_SECTIONS = {
   },
   "embedding-in-other-applications/full-app": {
     settings: [
-      {
-        key: "-breadcrumbs",
-        widget: () => {
-          return (
-            <Breadcrumbs
-              size="large"
-              crumbs={[
-                [
-                  t`Embedding`,
-                  "/admin/settings/embedding-in-other-applications",
-                ],
-                [t`Interactive embedding`],
-              ]}
-            />
-          );
-        },
-      },
-      {
-        key: "-link-widget",
-        widget: FullAppEmbeddingLinkWidget,
-        getHidden: (_, derivedSettings) =>
-          !derivedSettings["enable-embedding"] || PLUGIN_EMBEDDING.isEnabled(),
-      },
       {
         key: "-redirect-widget",
         widget: () => (
           <RedirectWidget to="/admin/settings/embedding-in-other-applications" />
         ),
-        getHidden: (_, derivedSettings) => derivedSettings["enable-embedding"],
+        getHidden: () => PLUGIN_EMBEDDING.isEnabled(),
```

Comment:
The plugin returns if the feature is in the features of the token, in that case we don't have to redirect otherwise EE users won't be able to access the page either

Reply:
I think this is incorrect. This change means that the settings page is now accessible even if embedding is off. We should still redirect users back to `/admin/settings/embedding-in-other-applications` if embedding is off (as if the setting page is inaccessible). I tested that reverting to the previous change works. Is there any reason you chose to do this in the first place?

![image](https://github.com/metabase/metabase/assets/1937582/31157fab-1318-459b-83de-fbc582398000)

Reply:
That was on EE right?
On OSS that page doesn't exist anymore and I tried to redirect them to the main settings page, but you're correct that I should handle the EE case when the setting is off too.
If I'm correct this should be the new logic
`should redirect = plugin is disabled || setting is off`
-> 
`getHidden = !(plugin is disabled || setting is off)`

I'll add tests (unit should be ok for this) and fix the logic, thanks for pointing that out

Reply:
Not sure if that or the version without negation would read better, but it's your call.

`getHidden: embedding plugin is enabled && embedding is on`